apps/api/src/common/helper/csv/formExport.ts

```diff
@@ -34,7 +34,7 @@ export enum FormExportColumnHeaders {
   currentEmployment = 'Current Employment Type',
   employmentCircumstance = 'Employment Circumstance',
 
-  additionalComments = 'Additional Comments',
+  nonClinicalJobTitle = 'Additional Comments',
```

Comment:
The ticket is a bit confusing, however given that the title of the field is still `Additional Comments`, I feel a bit weird changing the key.

Reply:
Yeah I was a bit conflicted, we should probably change the header now that the content of the field has changed right?

Reply:
Yeah, I'd double check with Erica on that one. It's possible that the intent of the field has changed, designs just haven't been update yet. 
I have a suspicion that they're asking for a field with two purposes though.